### PR TITLE
Fix link rendering in downstream Satellite

### PR DESCRIPTION
**Pull request by adamlazik1**
Currently:
image

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/proc_creating-a-report-template.adoc [hunk 1]
```diff
@@ -39,3 +31,15 @@ If this field remains undefined, the users receive a free-text field in which th
...
 . Click *Submit* to save your changes.
+
+.Additional resources
+
+* For more information about safe mode, see xref:Report_Template_Safe_Mode_{context}[].
+
+* For more information about writing templates, see xref:Template_Writing_Reference_{context}[].
+
+* For more information about macros you can use in report templates, see xref:Template_Macros_{context}[].
+
```
Comment (mjivraja) [inline guides/common/modules/proc_creating-a-report-template.adoc:42]: This should render the same format without empty lines.

**Review by mjivraja — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_creating-a-report-template.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/proc_creating-a-report-template.adoc]: Thanks, fixed.

File: guides/common/modules/proc_creating-a-report-template.adoc [hunk 2]
```diff
@@ -39,3 +31,12 @@ If this field remains undefined, the users receive a free-text field in which th
 . Click the *Location* tab and add the locations where you want to use the template.
 . Click the *Organizations* tab and add the organizations where you want to use the template.
 . Click *Submit* to save your changes.
+
+.Additional resources
+
```
Comment (mjivraja) [inline guides/common/modules/proc_creating-a-report-template.adoc:36]:
```suggestion
```
Even this?

Comment (adamlazik1) [reply guides/common/modules/proc_creating-a-report-template.adoc]: fixed.

**Comment by Lennonka** [comment]
Cherry-picked:
- bbc979ce7..b99b1d5a3  3.9 -> 3.9
- 09ebc0bee..16c87c816  3.8 -> 3.8
- ab40928de..b7f86e48f  3.7 -> 3.7
- 1729dc7c6..5e0001da2  3.6 -> 3.6
- 940deeb2c..811775511  3.5 -> 3.5
- 84f70337f..d03de57af  3.4 -> 3.4
- e5d5ca3f1..a8229f022  3.3 -> 3.3
- 36730ce80..1dbfb9a9b  3.2 -> 3.2
- 06aeb1780..99fc07b1c  3.1 -> 3.1

**Pull request merged**